client/app/dbaas/logs/detail/logs-order.service.js

```diff
@@ -0,0 +1,40 @@
+class LogsOrderService {
+    constructor (OvhApiDbaas, ServiceHelper) {
+        this.OvhApiDbaasLogsOrder = OvhApiDbaas.Logs().Order().Lexi();
+        this.ServiceHelper = ServiceHelper;
+        this.staticOffer = {
```

Comment:
I think if there's more static offers, I would have export it in a constant file.

Reply:
Ahh okay :)